quickwit/quickwit-indexing/src/actors/indexing_service.rs

```diff
@@ -315,7 +324,16 @@ impl IndexingService {
         let max_concurrent_split_uploads_merge =
             (self.max_concurrent_split_uploads - max_concurrent_split_uploads_index).max(1);
 
-        let params_fingerprint = index_config.indexing_params_fingerprint();
+        let params_fingerprint = indexing_params_fingerprint(&index_config, &source_config);
+        if let Some(expected_params_fingerprint) = expected_params_fingerprint {
+            if params_fingerprint != expected_params_fingerprint {
+                warn!(
+                    expected_fingerprint = expected_params_fingerprint,
+                    actual_fingerprint = params_fingerprint,
+                    "params fingerprint mismatch"
+                );
+            }
```

Comment:
This will happen when the params got updated after the plan was calculated but before it was applied. We now the pipeline is very likely to be short lived in that case because it should be restarted by the control plane on the next recalculation. Should we take another action here other than warn?

Reply:
This is expected so I don't think it's necessary. I think it's better to log:

1. on the control plane:
- when we detected a index or source config change
- when we issue a new (changed) indexing plan

2. on indexers:
- when we received a new (changed) indexing plan
- when we spawn new pipelines
- when we shutdown pipelines

If we do that correctly, we should be able to observe correctly the impact of updating an index or source config.

Thoughts?